cellprofiler/gui/__init__.py

```diff
@@ -105,11 +105,21 @@ def draw_item_selection_rect(window, dc, rect, flags):
     # might work in Cocoa
     #
     import wx
-    if (sys.platform != 'darwin' or
```

Comment:
the maxsize below was there in the hope that 3.0 would fix the way the Mac rendered listboxes using RendererNative, alas, the problem seems to be worse and the code to fix it is below. Used in the pipeline list control and in the file list in the Images module.